monkey/monkey_island/cc/services/version_update.py

```diff
@@ -61,6 +64,17 @@ def _check_new_version():
     @staticmethod
     def get_download_link():
         return VersionUpdateService.VERSION_SERVER_DOWNLOAD_URL % (
-            env_singleton.env.get_deployment(),
+            VersionUpdateService.get_deployment_file(),
             get_version(),
         )
+
+    @staticmethod
+    def get_deployment_file() -> Optional[str]:
```

Comment:
This function doesn't get the deployment file. It gets the deployment type (e.g. docker, windows, etc).